apps/webstore/src/app/shared/components/modals/add-to-basket-modal/add-to-basket-modal.component.scss

```
@@ -174,6 +175,58 @@
   }
 }
 
+.size-selection {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: end;
+  height: 300px;
+
+  & div {
+    border: 2px solid rgba($color: #000000, $alpha: 0.3);
+    background-color: white;
+    position: absolute;
+
+    .horizontal,
+    .vertical {
+      position: absolute;
+      font-size: 12px;
+      font-weight: 400;
+      font-family: 'Abel';
+      color: rgba(0, 0, 0, 0.4);
+    }
+
+    .horizontal {
+      top: 0px;
+      right: 14px;
+    }
+
+    .vertical {
+      top: 17px;
+      right: -10px;
```

```suggestion
      top: 23px;
      right: -6px;
```
the text was overlapping the border.